lib/rex/ui/interactive.rb

```diff
@@ -83,8 +83,13 @@ def interact(user_input, user_output)
       self.completed = true
     end
 
-    # Return whether or not EOF was reached
-    return eof
```

Comment:
This bit is the only concern I have. I looked at the code and I couldn't see anywhere that this result was actually relied on, but obviously I could be wrong! So I'd appreciate an expert opinion on this one.